app/models/subscription_package.rb

```diff
@@ -1,3 +1,15 @@
 class SubscriptionPackage < ActiveRecord::Base
   has_many :companies
-end
+
+  def self.get_packages
+    packages = SubscriptionPackage.all
+    # Currently there are no packages in db
+    # So packages can be blank therefore provided default packages
+    if packages.blank?
+      packages = [{ package_name: 'Platinum', max_supervisors: 50, max_members: 500 },
```

Comment:
please create objects of subscription_packages instead of only key value pairs

Reply:
handled using rake task add_packages